cylc/uiserver/data_store_mgr.py

```diff
@@ -133,7 +133,7 @@ async def sync_workflow(self, w_id, contact_data):
         )
         await self.entire_workflow_update(ids=[w_id])
 
-    async def register_workflow(self, w_id, name, owner):
```

Comment:
`name` and `owner` are no longer used and can be safely removed (looked at code using it, looks OK)